tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/BasicCasesTest.java

```diff
@@ -45,138 +46,141 @@ public void setup() throws BallerinaTestException, IOException {
+    public void testAssertionErrorMessage() throws BallerinaTestException, IOException {
         String[] args = mergeCoverageArgs(new String[]{"assertions-error-messages"});
         String output = balClient.runMainAndReadStdOut("test", args,
                 new HashMap<>(), projectPath, true);
-        AssertionUtils.assertForTestFailures(output, "assertion diff message failure");
+        AssertionUtils.assertOutput("BasicCasesTest-testAssertionErrorMessage.txt", output);
     }
 
     @Test
-    public void testAssertBehavioralTypes() throws BallerinaTestException {
+    public void testAssertBehavioralTypes() throws BallerinaTestException, IOException {
         String[] args = mergeCoverageArgs(new String[]{"assertions-behavioral-types"});
         String output = balClient.runMainAndReadStdOut("test", args,
                 new HashMap<>(), projectPath, true);
-        AssertionUtils.assertForTestFailures(output, "assertion failure for behavioral data types");
+        AssertionUtils.assertOutput("BasicCasesTest-testAssertBehavioralTypes.txt", output);
     }
 
     @Test
-    public void testAssertStructuralTypes() throws BallerinaTestException {
+    public void testAssertStructuralTypes() throws BallerinaTestException, IOException {
         String[] args = mergeCoverageArgs(new String[]{"assertions-structural-types"});
         String output = balClient.runMainAndReadStdOut("test", args,
                 new HashMap<>(), projectPath, true);
-        AssertionUtils.assertForTestFailures(output, "assertion failure for structural data types");
+        AssertionUtils.assertOutput("BasicCasesTest-testAssertStructuralTypes.txt", output);
     }
 
     @Test
-    public void testAssertSequenceTypes() throws BallerinaTestException {
+    public void testAssertSequenceTypes() throws BallerinaTestException, IOException {
         String[] args = mergeCoverageArgs(new String[]{"assertions-sequence-types"});
         String output = balClient.runMainAndReadStdOut("test", args,
                 new HashMap<>(), projectPath, true);
-        AssertionUtils.assertForTestFailures(output, "assertion failure for sequence data types");
+        AssertionUtils.assertOutput("BasicCasesTest-testAssertSequenceTypes.txt", output);
     }
 
     @Test
-    public void testAnnotationAccess() throws BallerinaTestException {
+    public void testAnnotationAccess() throws BallerinaTestException, IOException {
+        String endString = " SEVERE {b7a.log.crash} - ";
+        String firstString = "We thank you for helping make us better.";
         String[] args = mergeCoverageArgs(new String[]{"annotation-access"});
         String output = balClient.runMainAndReadStdOut("test", args,
                 new HashMap<>(), projectPath, true);
-        AssertionUtils.assertForTestFailures(output, "test annotation access failure");
+        output = CommonUtils.replaceVaryingString(firstString, endString, output);
```

Comment:
What is usually a varying string? The project path ? 
Can you please share an example?

Reply:
```[2023-02-22 16:38:10,019] SEVERE {b7a.log.crash} - class org.wso2.ballerinalang.compiler.semantics.model.symbols.BSymbol cannot be cast to class org.wso2.ballerinalang.compiler.semantics.model.symbols.BAnnotationSymbol (org.wso2.ballerinalang.compiler.semantics.model.symbols.BSymbol and org.wso2.ballerinalang.compiler.semantics.model.symbols.BAnnotationSymbol are in unnamed module of loader 'app') 
java.lang.ClassCastException: class org.wso2.ballerinalang.compiler.semantics.model.symbols.BSymbol cannot be cast to class org.wso2.ballerinalang.compiler.semantics.model.symbols.BAnnotationSymbol (org.wso2.ballerinalang.compiler.semantics.model.symbols.BSymbol and org.wso2.ballerinalang.compiler.semantics.model.symbols.BAnnotationSymbol are in unnamed module of loader 'app')
``` Here `[2023-02-22 16:38:10,019]` varies with time.

Reply:
Temporary directory Paths where test resource packages are copied to